packages/twenty-front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterDropdownFilterSelect.tsx

```diff
@@ -25,31 +55,50 @@ export const ObjectFilterDropdownFilterSelect = () => {
 
   const setHotkeyScope = useSetHotkeyScope();
 
+  const debouncedSetSearchFilter = useDebouncedCallback(setSearchText, 100, {
```

Comment:
I don't think we need a debounce for this as this is not querying/computing anything, let's not slow down the experience when it's not needed 👍